TestGeometricSeqPhenotype.java

```diff
@@ -0,0 +1,199 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.Random;
+import java.util.Scanner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * A class for testing the methods in GeometricSeqPhenotype.java
+ *
+ * @author Thien Tran
+ */
+public class TestGeometricSeqPhenotype {
+    // Private fields
+    private GeometricSeqPhenotype emptyPheno; // Default constructor
+    private GeometricSeqPhenotype simplePheno; // Give a specific sequence
+    private GeometricSeqPhenotype simplePheno2; // Give a specific sequence
+
+    GeometricSeqPhenotype [] history;
+
+    /**
+     * Define variables needed for multiple tests.
+     */
+    @Before
+    public void setUp() {
+        // initialize static parameters
+        Parameters.load();
+        Parameters.initialize();
+
+        Random r = new Random();
+        long seed = 5;
+        r.setSeed(seed);
+        emptyPheno = new GeometricSeqPhenotype();
```

Comment:
Could you please add a line or two of comments that describe what you're doing here, and what these objects will be used for?